include/wasm/WasmModule.h

```diff
@@ -121,7 +159,15 @@ class WasmModule
     int stdoutMemFd;
     ssize_t stdoutSize;
 
-    std::mutex moduleMemoryMutex;
+    faabric::util::Queue<threads::OpenMPTask> openMPTaskQueue;
+    faabric::util::Queue<int32_t> openMPResultQueue;
+    std::vector<std::thread> openMPThreads;
+    std::mutex threadsMutex;
+    std::mutex threadStacksMutex;
```

Comment:
These extra instance variables associated with the wasm module are unfortunate. Executing OpenMP threads is quite involved, accessing other internals of the wasm module class, so this was the easiest way to do it for now. I'd like to try and factor this out into a separate thread pool class eventually.